apps/mull-api/src/app/media/media.service.ts

```diff
@@ -65,6 +59,12 @@ export class MediaService {
     return await this.mediaRepository.save(newMedia);
   }
 
+  async deleteMedia(id: number): Promise<Media> {
+    const media = this.getMedia(id);
+    await this.mediaRepository.delete(id);
+    return media;
+  }
```

Comment:
Can this be a void function? I see the return data is not being used anywhere also if I understand correctly, it's returning the deleted media.
```suggestion
  async deleteMedia(id: number): Promise<void> {
    await this.mediaRepository.delete(id);
  }
```

Reply:
Ah I had asked Cristian and apparently returning the deleted entity was a common practice (it is seen in other service files)? I can still modify it however

Reply:
For most of our endpoints, we do return the object. For other cases, we also just return a boolean to confirm if the transaction went through

Reply:
Yeah I see we do it in our other service classes as well. This can be cleaned up in a maintenance ticket, returning **deleted** data is not a common practice for CRUD operations.